sites/public/src/components/directory/ListingDirectory.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useContext } from "react"
+import Head from "next/head"
+import { Heading } from "@bloom-housing/ui-seeds"
+import { Listing } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import { PageHeader, t } from "@bloom-housing/ui-components"
+import { ListingList, pushGtmEvent, AuthContext } from "@bloom-housing/shared-helpers"
+import { UserStatus } from "../../lib/constants"
+import Layout from "../../layouts/application"
+import { MetaTags } from "../../components/shared/MetaTags"
+import { ListingCard } from "./ListingCard"
+import styles from "./ListingDirectory.module.scss"
+
+export interface ListingDirectoryProps {
+  openListings: Listing[]
+  closedListings: Listing[]
+}
+
+export const ListingDirectory = (props: ListingDirectoryProps) => {
+  const { profile } = useContext(AuthContext)
+  const pageTitle = `${t("pageTitle.rent")} - ${t("nav.siteTitle")}`
+  const metaDescription = t("pageDescription.welcome", { regionName: t("region.name") })
+
+  useEffect(() => {
+    pushGtmEvent<ListingList>({
+      event: "pageView",
+      pageTitle: "Rent Affordable Housing - Housing Portal",
+      status: profile ? UserStatus.LoggedIn : UserStatus.NotLoggedIn,
+      numberOfListings: props.openListings.length,
+      listingIds: props.openListings.map((listing) => listing.id),
+    })
+  }, [profile, props.openListings])
+
+  return (
+    <Layout>
+      <Head>
+        <title>{pageTitle}</title>
+      </Head>
+
+      <MetaTags title={t("nav.siteTitle")} description={metaDescription} />
+      <PageHeader title={t("pageTitle.rent")} />
+
+      <div className={styles["listing-directory"]}>
+        <div className={styles["content-wrapper"]}>
+          {/* TODO: Show both open and closed listings once we have designs for pagination: Issue #4448 */}
+          <>
+            {props.openListings.length > 0 ? (
+              <ul>
```

Comment:
question: you mentioned this in the PR description so this is just for my own accessibility understanding, is an unordered list preferable to an ordered list for the screen reader?

Reply:
Neither is necessarily preferable to a screen reader / they're both great if relevant - it's just that the listing of listings doesn't have a logical top to bottom order, like say the list of numbered eligibility criteria does on the listing detail page that goes from most to least restrictive.